pkg/ui/workspaces/cluster-ui/src/databasesPage/util.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+import React from "react";
+import { CircleFilled } from "../icon";
+import { DatabasesPageDataDatabase } from "./databasesPage";
+import classNames from "classnames/bind";
+import styles from "./databasesPage.module.scss";
+import { EncodeDatabaseUri } from "../util";
+import { Link } from "react-router-dom";
+import { StackIcon } from "../icon/stackIcon";
+
+const cx = classNames.bind(styles);
+
+export const IndexRecCell = ({
+  database,
+}: {
+  database: DatabasesPageDataDatabase;
+}): JSX.Element => {
+  const text =
+    database.numIndexRecommendations > 0
+      ? `${database.numIndexRecommendations} index ${
+          database.numIndexRecommendations > 1
+            ? "recommendations"
+            : "recommendation"
+        }`
+      : "None";
+  const classname =
+    database.numIndexRecommendations > 0
+      ? "index-recommendations-icon__exist"
+      : "index-recommendations-icon__none";
+  return (
+    <div>
+      <CircleFilled className={cx(classname)} />
+      <span>{text}</span>
+    </div>
+  );
+};
+
+export const DatabaseNameCell = ({
+  database,
+  isCockroachCloud,
+}: {
+  database: DatabasesPageDataDatabase;
+  isCockroachCloud: boolean;
```

Comment:
You can use this hook directly in the component rather than passing the prop here.

Reply:
Done.